data-raw/weather.R

```diff
@@ -66,8 +66,11 @@ weather <- raw %>%
   ungroup() %>%
   filter(!is.na(month)) %>%
   mutate(
-    time_hour = ISOdatetime(year, month, day, hour, 0, 0)
-  )
+    time_hour = ISOdatetime(year, month, day, hour, 0, 0),
```

Comment:
Are you sure you can't just set `tz` here? 

Your current code doesn't feel right to me - you shouldn't have to both force the timezone and then add a manual offset